### PR TITLE
[16.0][MIG] sale_financial_risk_info: Migration to version 16.0

### PR DESCRIPTION
**Pull request by ernesto-garcia-tecnativa**
@tecnativa TT44215

### CODE REVIEW

**Comment by ernesto-garcia-tecnativa** [comment]
@pedrobaeza @carlosdauden please review

**Comment by pedrobaeza** [comment]
You have mixed the changes for the migration in the pre-commit commit. Please fix it.

**Comment by ernesto-garcia-tecnativa** [comment]
done @pedrobaeza

File: sale_financial_risk_info/tests/test_sale_financial_risk_info.py [hunk 1]
```diff
@@ -5,44 +5,34 @@
...
-        self.invoice_partner_1 = self.create_invoice(self.partner_1)
+        cls.invoice_partner_1 = cls.create_invoice(cls.partner_1)
 
     # Create some invoices for partner
-    def create_invoice(self, partner):
+    @classmethod
+    def create_invoice(cls, partner):
         move_form = Form(
-            self.env["account.move"].with_context(default_type="out_invoice")
+            cls.env["account.move"].with_context(default_move_type="out_invoice")
```
Comment (pedrobaeza) [inline sale_financial_risk_info/tests/test_sale_financial_risk_info.py]: In v16, it's even desirable to not use `Form` for invoices. Check https://github.com/odoo/odoo/blob/f73c214efdbd49865ff3f0db915fadc80b4403b0/addons/account/tests/common.py#L674

**Review by BT-anieto — APPROVED** [review]
LGTM

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**